### PR TITLE
Update RULES_ENGINE.md

### PR DESCRIPTION
**Pull request by Dorota-MB**

### CODE REVIEW

File: RULES_ENGINE.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Conditions can be defined with either "all" or "any" type of arrays. In each arr
   * Fact - facts are the variable names that can be one of the following
     * path - The request path
   
-      Example: /transfers, /parties/MSISDN/9876543210 ... etc
+      Example: /transfers, /parties/MSSDN/9876543210 ... etc
```
Comment (elnyry) [inline RULES_ENGINE.md]: MSISDN is correct

File: RULES_ENGINE.md [hunk 2]
```diff
@@ -228,13 +228,13 @@ The following is the list of configurable parameters that you can define anywher
...
-There are some advanced parameters type *$function* is implemented for advanced users / developers. The syntax is like below
+There are some advanced parameters type *$function* for advanced users / developers. Here is the exammple syntax:
 
 * $function.sample.getSampleText (For development purpose)
 
-The above parameter calls the function 'getSampleText' in 'sample.js' file located at 'lib/mocking/custom-functions' folder. An argument will be passed to this function from which the developer can get the request parameters to process like 'arg1.request.body.quoteId', 'arg1.request.headers.date' ...etc
+The above parameter calls the function 'getSampleText' in 'sample.js' file located at 'lib/mocking/custom-functions' folder. An argument will be passed to this function from which the developer can get the request parameters to process, for example 'arg1.request.body.quoteId', 'arg1.request.headers.date' ...etc
 
-Please be noted that these javascript files can not be change in runtime and need compilation for the changes.
+Please note that these javascript files can not be changed at runtime and need to be compilated prior to running.
```
Comment (elnyry) [inline RULES_ENGINE.md]: 'compiled'

File: RULES_ENGINE.md [hunk 3]
```diff
@@ -277,11 +277,11 @@ See the following example in which the generated values of the firstName & name
 
 ### Priority of rules
 
-The 'priority' property in a rule dictates when rule should be run, relative to other rules. Higher priority rules are run before lower priority rules. Rules with the same priority are run in parallel. Priority must be a positive, non-zero integer.
+The 'priority' property in a rule dictates when rule should be run, relative to other rules. Higher priority rules are run before lower priority rules. Rules with the same priority are ran in parallel. Priority must be a positive, non-zero integer.
```
Comment (elnyry) [inline RULES_ENGINE.md]: 'run' is fine here

File: RULES_ENGINE.md (hunk 1 above)
Comment (Dorota-MB) [reply RULES_ENGINE.md]: Sorry misspelled

**Pull request closed without merging**